pkg/registry/minion/rest.go

```diff
@@ -107,7 +107,8 @@ type ResourceGetter interface {
 // NodeToSelectableFields returns a label set that represents the object.
 func NodeToSelectableFields(node *api.Node) fields.Set {
 	return fields.Set{
-		"metadata.name": node.Name,
+		"metadata.name":      node.Name,
+		"spec.unschedulable": fmt.Sprint(node.Spec.Unschedulable),
```

Comment:
Ugh, did we really call this field Unschedulable? :( :( field names should always be in the positive IMO (should be Schedulable)